### PR TITLE
Use curl to download systemtest prereqs when on zos 

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Fixes: https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/127
Signed-off-by: Mesbah <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested on zOS with: https://urv-jenkins.canlab.ibm.com:9095/view/SV_Tests/job/ibmjava8_systemtest_s390x_zos/69/console (please disregard the error in the build, it happens at compile time, for a different issue).

Tested on non-Zos with: https://urv-jenkins.canlab.ibm.com:9095/view/SV_Tests/job/ibmjava8_systemtest_x86-64_linux/442/console

File: stf.build/include/top.xml [hunk 1]
```diff
@@ -114,7 +114,7 @@ limitations under the License.
 		Targets to check for and download prereqs.
 		All prereqs are installed under the same ${prereqs_root} directory.
 	-->
-	<target name="configure" depends="check-for-wget,
+	<target name="configure" depends="check-for-download-tool,
```
Comment (sophia-guo) [inline stf.build/include/top.xml:117]: Is there check-for-download-tool target already?

Comment (Mesbah-Alam) [reply stf.build/include/top.xml:117]: Yes, `check-for-download-tool` target is part of this pr.

Comment (sophia-guo) [reply stf.build/include/top.xml:117]: yeah, seems I didn't finish the whole PR.

File: stf.build/include/top.xml [hunk 2]
```diff
@@ -235,16 +235,25 @@ limitations under the License.
...
 		<unzip src="${java.io.tmpdir}/windows_sysinternals/SysinternalsSuite.zip" dest="${windows_sysinternals_dir}"/>
 		<delete dir="${java.io.tmpdir}/windows_sysinternals"/>
 		<!-- We're just using the existence of pskill.exe to determine whether the tools are installed. -->
 	    <available file="${windows_sysinternals_dir}/pskill.exe" property="windows_sysinternals_available"/>
 	</target>
 
+	<target name="check-for-download-tool" depends="check-for-wget, check-for-curl">
+		<condition property="downloadtool_available" value="true">
+			<or>
+				<isset property="wget_available"/>
```
Comment (sophia-guo) [inline stf.build/include/top.xml]: Is it possible that zOS do have wget installed?

**Review by sophia-guo — APPROVED** [review]

**Pull request merged**